ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -192,15 +194,22 @@ - (void)requestCode:(ADAuthorizationCodeCallback)completionBlock
                  {
                      NSDictionary* queryParams = [end adQueryParameters];
                      code = [queryParams objectForKey:OAUTH2_CODE];
+                     [self setCloudInstanceHostname:[queryParams objectForKey:AUTH_CLOUD_INSTANCE_HOST_NAME]];
                  }
                  else
                  {
-                     NSDictionary* userInfo = @{
-                                                @"username": [[NSDictionary adURLFormDecode:[end query]] valueForKey:@"username"],
-                                                };
+                     NSMutableDictionary *userInfoDictionary = [NSMutableDictionary dictionary];
+                     NSDictionary *queryParameters = [NSDictionary adURLFormDecode:[end query]];
+                     NSString *userName = [queryParameters valueForKey:AUTH_USERNAME_KEY];
+                     
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)